hooks/after_prepare/010_add_platform_class.js

```diff
@@ -37,8 +37,15 @@ function addPlatformBodyTag(indexPath, platform) {
       newBodyTag = bodyTag.replace(classAttr, newClassAttr);
 
     } else {
+
+      /*
+       The bodyTag contains only the first found of `>`, due to the return of the first element of the array from method
+       (findBodyTag)[https://github.com/xlts-dev/angularjs-ionic-v1/blob/c553c973d85142e66f119aed2fe3db83775f5f2b/hooks
+       /after_prepare/010_add_platform_class.js#L57]. Therefore, we should replace only one tag
+       */
```

Comment:
Nit: This comment feels a bit redundant since we are doing the obvious thing. (Comments are more useful if the code is doing something complex or unintuitive/non-obvious.)

If we keep it though, it would be better to use a `//` comment to match the surrounding style.

Reply:
A difference between `/* */` and `//` comments?
No (except for the obvious one that the former can span multiple lines :grin:).

A difference in following the surrounding style?
Yes :smiley:
Following a consistent code style makes the code more readable :wink: